data/beps.yml

```diff
@@ -92,6 +99,7 @@
     link: https://docs.google.com/document/d/1ugBdUF6dhElXdj3u9vw0iWjE6f_Bibsro3ah7sRV0GA/
     leads:
     -   name: Eugene Duff
+        email: eugene.duff@paediatrics.ox.ac.uk
```

Comment:
@eduff This email seems dead. Are you still a BEP lead on BEP17? How should we contact you?

Reply:
eugene.duff@gmail.com

Reply:
Yes I'm a co-lead.